src/index.js

```diff
@@ -143,16 +143,24 @@ export const getTargets = (targets = {}) => {
   return targetOps;
 };
 
+export const validateBoolOption = (name, value, defaultValue) => {
+  if (typeof value === "undefined") {
```

Comment:
Seems like a good place to use Lodash's `isUndefined` and `isBoolean` since it's already in the package.json.